src/components/footer/sections/footer-contact.tsx

```diff
@@ -0,0 +1,36 @@
+import '@/tw-styles.css';
+
+export default function Contact () {
+    return (
+        <>
+            <div className="w-full flex justify-center border-t border-t-[#aaa]">
+                <div className="w-[1080px] py-[15px] flex justify-between items-center title-b-16">
+                    <div className="flex gap-[30px] text-[var(--black)]">
```

Comment:
```suggestion
                    <div className="flex gap-[30px] text-black">
```

변수화 했으니 이렇게 사용하시면 됩니다!

Reply:
넵 그렇게 사용하겠습니다!